_rules/disable-shortcut-670a30.md

```diff
@@ -0,0 +1,337 @@
+      <label>
+        <input id="turn_off" type="checkbox">
+      </label>
+    </div>
+    <div id="text">Some text inside the document content</div>
+  </body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The [HTML document][] does not use [keyboard shortcuts][keyboard shortcut].
+
+```html
+<html>
+  <div>Document content</div>
+</html>
+```
+
+#### Inapplicable Example 2
+
+The document is not an [HTML document][].
+
+```html
+<svg height="100" width="100">
+  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
+  Sorry, your browser does not support inline SVG.  
+</svg>
+```
+
+#### Inapplicable Example 3
+
+The [HTML document][] has a [keyboard shortcut][] that requires pressing one [non-printable character][non-printable characters].
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 3 for rule 670a30</title>
+    <script>
+      function shortcut(params) {
+        document.body.addEventListener("keydown", function(event) {
+          const target = document.getElementById("target");
+
+          if (
+            event.key === params.shortcutKey &&
+            (!!!params.ctrlKey || event.ctrlKey) &&
+            (!!!params.focusOnly || document.activeElement === target)
+          ) {
+            document.getElementById("list").innerHTML +=
+              "<li>" + target.value + "</li>";
+            target.value = "";
+            event.preventDefault();
+          }
+        });
+      }
+    </script>
+  </head>
+
+  <body onload="shortcut({focusOnly: true, shortcutKey: '+' , ctrlKey: true})">
+    <label for="text">Add to list (press ctrl and + to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Inapplicable Example 4
+
+The [HTML document][] has an element with the attribute `accesskey`. Accesskeys use [non-printable characters][].
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 4 for rule 670a30</title>
+    <script>
+      function shortcut(params) {
+        const button = document.querySelector("button");
+
+        button.addEventListener("click", function() {
+          document.getElementById("list").innerHTML +=
+            "<li>Item</li>";
+          event.preventDefault();
+        });
+
+        button.textContent = button.accessKeyLabel
+          ? "Add item to list (" + button.accessKeyLabel + ")"
+          : "Add item to list (accesskey +" + button.accessKey + ")";
+      }
+    </script>
+  </head>
+
+  <body onload="shortcut()">
+    <button type="button" accesskey="+"></button>
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+[HTML document]: https://dom.spec.whatwg.org/#concept-document
+[keyboard shortcut]: https://www.w3.org/TR/WCAG21/#dfn-keyboard-shortcuts
+[mechanism]: https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-mechanism
```

Comment:
I think the term is not used any more in the rule and can be removed (same in other rules).